src/enftun.c

```diff
@@ -47,10 +49,10 @@ static
 void
 stop_all(struct enftun_context* ctx)
 {
+    enftun_conn_state_stop(&ctx->conn_state);
```

Comment:
See my other comment - any `_stop` here should be paired with a `_start` in the `start_all`.